Rakefile

```diff
@@ -19,7 +21,7 @@ rescue LoadError
 end
 
 desc 'Run tests in generated test Rails app with generated Solr instance running'
-task ci: [:environment] do
+task ci: [:rubocop, :environment] do
```

Comment:
I really like having a task to run that spins up solr & runs tests without doing rubocop. But I can fix that in another PR.